deploy/examples/cluster-test.yaml

```diff
@@ -36,6 +36,12 @@ spec:
   mgr:
     count: 1
     allowMultiplePerNode: true
+  network:
```

Comment:
Is the plan to enable this once Quincy is out? Is this why we have it in cluster-test?